clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/sagas/AbstractSagaAtomicOperation.java

```diff
@@ -64,7 +64,7 @@ public AbstractSagaAtomicOperation(T description) {
    * Provides the opportunity to convert a {@link SagaAction.Result} into the expected result type
    * of the AtomicOperation.
    */
-  protected abstract R parseSagaResult(@Nonnull SR result);
+  protected abstract R parseSagaResult(SR result);
```

Comment:
This can totally be null right? (I think more often than not it's actually going to be null)

Reply:
Ah, yeah. That can be null, yes.